spec/integration/not_modified_response_spec.rb

```diff
@@ -115,7 +115,7 @@
     context "when requesting an unmodified response" do
       let(:path)         { "/components/batch" }
       let(:content_type) { "application/json" }
-      let(:etag)         { '"34774567db979628363e6e865127623f"' }
+      let(:etag)         { '"8d2f79a98b0e0c1fbcca34850f2c7f17"' }
```

Comment:
I know it was like this before, but this seems a bit odd using single and double quotes.

Reply:
Why have all the etag headers changed?

Reply:
its using the new version of crimp

Reply:
the quotes are unavoidable: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag#Directives

Reply:
All other headers are just strings, but as the spec says the ETag ones are returned with double quotes, so ours match the specification.